### DISCUSSION TITLE
Access granted to: undefined in logs

### DISCUSSION THREAD

**Discussion by Viiprogrammer** [verdaccio/verdaccio#3736] [Ideas]
Logs has "auth/allow_action: access granted to: undefined"

![image](https://user-images.githubusercontent.com/17622604/232309870-a98199d3-858f-4fa6-b744-f76b2877c57c.png)

I installing 2 packages:
- 1 non-exists on local registry
- 2 exists on local registry

`verdaccio` installed via `npm: npm i verdaccio -g` with zero config run from terminal. Everything works fine, but I think that undefined in the logs is not good behavior.

After install, one user added via `adduser`

**OS:** `Ubuntu 22.04`
**NPM:** `8.15.0`
**Node:** `16.17.0`
**verdaccio:** `5.23.2`

**Comment by juanpicado** [comment]
Fixed here https://github.com/verdaccio/verdaccio/commit/268c5e7740115fdef56fddca04ac2a4d3e5481d4

**Reply by Viiprogrammer** [reply]
Bug regression, now I see it is `undefined` for every HTTP interface page loading 🧐

https://user-images.githubusercontent.com/17622604/234033568-7faf9dce-f5c7-4f9f-bda3-ef9c4ac7623c.mp4

```bash
 righthaze@anime-desktop  ~  npm list -g
/home/righthaze/.nvm/versions/node/v16.17.0/lib
├── pnpm@7.22.0
├── verdaccio@5.24.1
└── yarn@1.22.19
```
